backend/src/data/repositories/space/space.repository.ts

```diff
@@ -45,6 +45,19 @@ class Space {
     return Space.modelToEntity(space as SpaceM);
   }
 
+  public async getByWorkerId(workerId: string): Promise<string[]> {
+    const space = await this.#SpaceModel
+      .query()
+      .select()
+      .where({ 'createdBy': workerId });
+
+    if (!space) {
+      return [];
+    }
+
+    return space.map((item) => item.id);
+  }
```

Comment:
Same as with slc functions and instances

Reply:
+